pos_barcode_sync/models/pos_sync_barcodes_models.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, api
+from odoo.http import request
+
+
+def send_barcode_update(message, partner_ids):
+    channel_name = "pos_barcode_sync"
+    pos_configs = request.env["pos.config"].search([('active', '=', True)])
+    res = pos_configs._send_to_channel(channel_name, {'message': message, 'partner_ids': partner_ids})
+    return res
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
```

Comment:
Follow odoo guidelines for FILE naming please
https://www.odoo.com/documentation/10.0/reference/guidelines.html